How to run WooCommerce core tests?

I'm trying, without luck, to run the tests in `node_modules/@woocommerce/e2e-core-tests/specs` directory. i.e. I want to confirm my plugin is not breaking anything in core. I have what should be a failing test – a plugin where I was hooking too early and could no longer change product images ([a related issue](https://wordpress.org/support/topic/amzon-pay-pluggin-conflict/)).

I've toyed around with the Jest config, in particular removing the default exclusion on `node_modules` (`NODE_MODULES_REGEXP`) and adding the path to the core tests:

```
const jestConfig = useE2EJestConfig( {
	roots: [ path.resolve( __dirname, '../specs' ), path.resolve( __dirname, '../../../node_modules/@woocommerce/e2e-core-tests/specs' ) ],
	testMatch: [ '**/*.(test|spec).js', '*.(test|spec).js', 'index.js'],
	testPathIgnorePatterns: []
} );
```

but I still get `No tests found`:

```
npm  run test:e2e node_modules/@woocommerce/e2e-core-tests/specs
```
```
> bh-wc-set-gateway-by-url@1.0.6 test:e2e /Users/brianhenry/Sites/bh-wc-set-gateway-by-url
> npx wc-e2e test:e2e "node_modules/@woocommerce/e2e-core-tests/specs"

Testing URL: http://localhost:8084

No tests found, exiting with code 1
Run with `--passWithNoTests` to exit with code 0
No files found in /Users/brianhenry/Sites/bh-wc-set-gateway-by-url/node_modules/@woocommerce/e2e-environment.
Make sure Jest's configuration does not exclude this directory.
To set up Jest, make sure a package.json file exists.
Jest Documentation: facebook.github.io/jest/docs/configuration.html
Pattern: node_modules/@woocommerce/e2e-core-tests/specs - 0 matches
Jest exit code: 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! bh-wc-set-gateway-by-url@1.0.6 test:e2e: `npx wc-e2e test:e2e "node_modules/@woocommerce/e2e-core-tests/specs"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the bh-wc-set-gateway-by-url@1.0.6 test:e2e script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/brianhenry/.npm/_logs/2021-07-09T17_10_51_761Z-debug.log
```

In the future, I hope all plugins on the WooCommerce store will be obliged to publish their tests so we can run them with our own plugins active.

--- a/README.md
+++ b/README.md
@@ -102,6 +102,8 @@ const { runActivationTest } = require( '@woocommerce/e2e-core-tests' );
 runActivationTest();
 ```
 
+For more imformation about the core tests, including what tests are available, how to re-run tests, and general setup, please see the [WooCommerce Core End to End Test Suite README](https://github.com/woocommerce/woocommerce/blob/trunk/tests/e2e/core-tests/README.md).
+
 ### Relevant files
 
 * **Docker Initialization Script** - `tests/e2e/docker/initialize.sh`. This can be used to set up your testing environment such as installing additional plugins via WP-CLI / importing data / running additional scripts. Reference - https://github.com/woocommerce/woocommerce/blob/master/tests/e2e/docker/initialize.sh

--- a/README.md
+++ b/README.md
@@ -92,6 +92,16 @@ To add new E2E tests, add new files to **`tests/e2e/specs/`**. **`tests/e2e/spec
 - Files with the extensions **`.test.js`** or **`.spec.js`** are run automatically by `npm run test:e2e`.
 - Adhoc tests can be created with the **`.js`** and executed with `npm run test:e2e tests/e2e/specs/test-example.js`.
 
+### Running core tests
+
+If you're installed the `@woocommerce/e2e-core-tests` and wish to run the tests within your project, note that all of the tests are wrapped in functions. In order to run the tests, create a local test spec and use the test function:
+
+```javascript
+const { runActivationTest } = require( '@woocommerce/e2e-core-tests' );
+
+runActivationTest();
+```
+
 ### Relevant files
 
 * **Docker Initialization Script** - `tests/e2e/docker/initialize.sh`. This can be used to set up your testing environment such as installing additional plugins via WP-CLI / importing data / running additional scripts. Reference - https://github.com/woocommerce/woocommerce/blob/master/tests/e2e/docker/initialize.sh

--- a/README.md
+++ b/README.md
@@ -94,7 +94,7 @@ To add new E2E tests, add new files to **`tests/e2e/specs/`**. **`tests/e2e/spec
 
 ### Running core tests
 
-If you're installed the `@woocommerce/e2e-core-tests` and wish to run the tests within your project, note that all of the tests are wrapped in functions. In order to run the tests, create a local test spec and use the test function:
+If you installed the optional `@woocommerce/e2e-core-tests` package and wish to run the tests within your project, note that all of the tests are wrapped in functions. In order to run the tests, create a local test spec and use the test function:
 
 ```javascript
 const { runActivationTest } = require( '@woocommerce/e2e-core-tests' );

--- a/README.md
+++ b/README.md
@@ -94,7 +94,7 @@ To add new E2E tests, add new files to **`tests/e2e/specs/`**. **`tests/e2e/spec
 
 ### Running core tests
 
-If you installed the optional `@woocommerce/e2e-core-tests` package and wish to run the tests within your project, note that all of the tests are wrapped in functions. In order to run the tests, create a local test spec and use the test function:
+If you installed the optional `@woocommerce/e2e-core-tests` package and wish to run the tests within your project, note that all of the tests are wrapped in functions. In order to run the tests, create a local test spec and use the test function. For example, to run the activation tests, you would need to do the following:
 
 ```javascript
 const { runActivationTest } = require( '@woocommerce/e2e-core-tests' );

